js/components/styles.js

```diff
@@ -66,7 +66,31 @@ const ProfileStyles = StyleSheet.create({
   }
 })
 
+const PortfolioStyles = StyleSheet.create({
+  titleStyle: {
+    fontFamily: 'Titillium Web',
```

Comment:
this must come from some constant (otherwise if we'll need to change fonts we have to search-and-replace all files).
this states for colors and sizes as well. @fpersico what's the best practice here ?